README.md

```diff
@@ -1,2 +1,85 @@
 # capstone
 Capstone project for Module 1 At Microverse
+
+
+
+# HTML/CSS & JavaScript capstone project - Web Development Course page
+
+> A web app to showcase web development course with all the contents and instructors.
+
+## Built With
+
+- HTML, CSS & JS
+- Hint, Stylelint, Eslint
+
+## Live Demo (if available)
+
+[Web Development Course]
+(https://sangwa7.github.io/capstone/index.html)
```

Comment:
[OPTIONAL]
This link will be displayed as intended if you remove the space between the closing square bracket and the parenthesis.

- [ ] Please take into consideration to remove that space which is redundant.